packages/examples/graphiql-webpack/webpack.config.js

```diff
@@ -0,0 +1,70 @@
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+const path = require('path');
+const isDev = process.env.NODE_ENV === 'development';
+
+module.exports = {
+  entry: isDev
+    ? [
+        'react-hot-loader/patch', // activate HMR for React
+        'webpack-dev-server/client?http://localhost:8080', // bundle the client for webpack-dev-server and connect to the provided endpoint
+        'webpack/hot/only-dev-server', // bundle the client for hot reloading, only- means to only hot reload for successful updates
+        './index.jsx', // the entry point of our app
+      ]
+    : './index.jsx',
+  context: path.resolve(__dirname, './src'),
+  mode: 'development',
+  devtool: 'inline-source-map',
+  performance: {
+    hints: false,
+  },
+  module: {
+    rules: [
+      {
+        test: /\.html$/,
+        use: ['file?name=[name].[ext]'],
+      },
+      // for graphql module, which uses mjs still
+      {
+        type: 'javascript/auto',
+        test: /\.mjs$/,
+        use: [],
+        include: /node_modules/,
+      },
+      {
+        test: /\.(js|jsx)$/,
+        use: [
+          {
+            loader: 'babel-loader',
+            options: {
+              presets: [
+                ['@babel/preset-env', { modules: false }],
+                '@babel/preset-react',
+              ],
+            },
+          },
+        ],
+      },
+      {
+        test: /\.css$/,
+        use: ['style-loader', 'css-loader'],
+      },
+      {
+        test: /\.svg$/,
+        use: [{ loader: 'svg-inline-loader' }],
+      },
+    ],
+  },
+  resolve: {
+    extensions: ['.js', '.json', '.jsx', '.css', '.mjs'],
```

Comment:
OPTIONAL: Having justification for this search order in a comment would be handy.

Reply:
yes, we will address this when we are bundling graphiql's min bundle with webpack in the next few rounds